front/app/modules/commercial/content_builder/admin/components/ContentBuilderSettings/index.tsx

```diff
@@ -50,7 +50,7 @@ const ContentBuilderSettings = () => {
       position="fixed"
       right="0"
       top={`${stylingConsts.menuHeight * 2}px`}
-      zIndex="2"
+      zIndex="99999"
       p="20px"
```

Comment:
The problem with the green indicator showing on top of the Settings panel is because the indicator has a really high z-value which is hardcoded in the craftjs library. Using this new z value ensures the panels will always display on top.